oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profile/analyze/ProfileAnalyzer.java

```diff
@@ -50,9 +133,28 @@ public static ProfileAnalyzation analyze(List<ProfileStack> stacks) {
                 .filter(s -> CollectionUtils.isNotEmpty(s.getStack()))
                 .collect(Collectors.groupingBy(s -> s.getStack().get(0), ANALYZE_COLLECTOR));
 
-        ProfileAnalyzation analyzer = new ProfileAnalyzation();
-        analyzer.setTrees(new ArrayList<>(stackTrees.values()));
-        return analyzer;
+        return new ArrayList<>(stackTrees.values());
     }
 
+    private IProfileThreadSnapshotQueryDAO getProfileThreadSnapshotQueryDAO() {
+        if (profileThreadSnapshotQueryDAO == null) {
+            profileThreadSnapshotQueryDAO = moduleManager.find(StorageModule.NAME).provider().getService(IProfileThreadSnapshotQueryDAO.class);
+        }
+        return profileThreadSnapshotQueryDAO;
+    }
+
+    private static class SequenceSearch {
+        LinkedList<SequenceRange> ranges = new LinkedList<>();
+        int totalSequenceCount;
+    }
+
+    private static class SequenceRange {
```

Comment:
Even for an inner class, please use set/get, rather than accessing fields directly.